lib-multisrc/colamanga/src/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt

```diff
@@ -164,13 +167,16 @@ abstract class ColaManga(
             statusCompleted -> SManga.COMPLETED
             else -> SManga.UNKNOWN
         }
+        val formatter: SimpleDateFormat = SimpleDateFormat("yyyy-MM-dd")
+        lastUpdatedDate = formatter.parse(document.selectFirst("span.fed-text-muted:contains($lastUpdated) + a")?.text()).getTime()
     }
```

Comment:
this would only set a date when user refresh chapter list manually, and this can have race conditions on library update too. Besides, I don't think setting date of each chapter to last upload is a good idea, maybe only set the latest chapter date?

Better to parse and set it `chapterListParse` (or `chapterFromElement`)
also save the `SimpleDateFormat` in class as val to avoid recreating it each time.